BitwardenShared/UI/Auth/IntroCarousel/IntroCarouselProcessor.swift

```diff
@@ -22,22 +29,45 @@ class IntroCarouselProcessor: StateProcessor<IntroCarouselState, IntroCarouselAc
     ///
     init(
         coordinator: AnyCoordinator<AuthRoute, AuthEvent>,
+        services: Services,
         state: IntroCarouselState
     ) {
         self.coordinator = coordinator
+        self.services = services
         super.init(state: state)
     }
 
     // MARK: Methods
 
+    override func perform(_ effect: IntroCarouselEffect) async {
+        switch effect {
+        case .createAccount:
+            let isEmailVerificationEnabled: Bool = await services.configService.getFeatureFlag(
+                .emailVerification,
+                isPreAuth: true
+            )
+            if isEmailVerificationEnabled {
+                coordinator.navigate(to: .startRegistration, context: self)
```

Comment:
❓ should we also be checking the `native-create-account-flow` flag or is that only required if you come into the flow from the log in screen?

Reply:
oh yeah, thats right... my b!